torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6380,17 +6380,21 @@ def sample_inputs_spectral_ops(self, device, dtype, requires_grad=False, **kwarg
         # cuFFT supports powers of 2 for half and complex half precision
         # NOTE: For hfft, hfft2, hfftn, irfft, irfft2, irfftn with default args
         # where output_size n=2*(input_size - 1), we make sure that logical fft size is a power of two
+        low = None
+        high = None
         if self.name in ['fft.hfft', 'fft.irfft']:
             shapes = ((2, 9, 9), (33,))
         elif self.name in ['fft.hfft2', 'fft.irfft2']:
             shapes = ((2, 8, 9), (33,))
         elif self.name in ['fft.hfftn', 'fft.irfftn']:
             shapes = ((2, 2, 33), (33,))
+            low = -1.0
```

Comment:
Add a comment for why this occurs.

Another option, instead of any of this, is to just increase the absolute tolerance for the test by applying a toleranceOverride decorator using the OpInfo. It would just apply for the chalf datatype, and avoid mucking around with the sample inputs. I would prefer that approach because it would be clearer about what's going on.

Reply:
We already have such relaxation for other test (test_reference_nd).

https://github.com/pytorch/pytorch/blob/dd73c97ea281129ee690b2131d8e8cde3a669285/torch/testing/_internal/common_methods_invocations.py#L11892-L11896

Reply:
> Another option, instead of any of this, is to just increase the absolute tolerance for the test by applying a toleranceOverride decorator using the OpInfo. It would just apply for the chalf datatype, and avoid mucking around with the sample inputs. I would prefer that approach because it would be clearer about what's going on.

I was having the same idea in one of my earlier commits. https://github.com/pytorch/pytorch/pull/81416/commits/5107a8fe421cc78f14422170e3e6b66904447dbd I can restore it if everyone is happy about it.

Reply:
> I would prefer that approach because it would be clearer about what's going on.

I'm going to disagree here. I think this is preferable to increasing tolerance because we test the precision of the implementation, not just the point where float16 saturates.

Reply:
> > I would prefer that approach because it would be clearer about what's going on.
> 
> I'm going to disagree here. I think this is preferable to increasing tolerance because we test the precision of the implementation, not just the point where float16 saturates.

OK -- let's go with this approach, then